department-of-veterans-affairs/caseflow
# Queue | Set default userRole

Connects #5905 

### Description
Some users have a `vacols_role` of `nil`. With the [rollout of the `:case_search_home_page`](https://github.com/department-of-veterans-affairs/caseflow/blob/9479f7d98142d719398e104215316a68499b224b/app/controllers/home_controller.rb#L7) FeatureToggle, we allow users to access Queue who have no `vacols_role`. However, the site wouldn't work for these users, because within `QueueApp`, we [use the user role to set the page title](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/queue/QueueApp.jsx#L166).

### Acceptance Criteria 
- [ ] User with no `vacols_role` can load case search home page

### Testing Plan
1. Enable `:case_search_home_page` FeatureToggle
2. Go to `/` as `BVARROHAN`, confirm case search home page renders/works as expected

Screenshot shows page loading, `userRole` attr from server is `""` if no `vacols_role`
<img width="670" alt="screen shot 2018-06-13 at 2 57 12 pm" src="https://user-images.githubusercontent.com/8533004/41372010-189aa16a-6f1a-11e8-864e-14925ec2ee92.png">
